### ISSUE TITLE
No support for pacman

### ISSUE THREAD

**Issue by ekarnis** [idank/explainshell#197]
I typed in sudo pacman -Syu git and it evaluated pacman as a user rather than a command.

https://explainshell.com/explain?cmd=sudo+pacman+-Syu+git

**Comment by stoivo** [comment]
What are you asking? 
Are you just reporting that packman is missing?

**Comment by ekarnis** [comment]
Yes, though after I opened this issue I suddenly understood that the site was only for shell commands, which I should have understood before. Feel free to close the issue if you like.
